eng/sendtohelix.proj

```diff
@@ -82,7 +84,33 @@
 
   </Target>
 
-  <Target Name="BuildHelixWorkItems" DependsOnTargets="CompressRuntimeDirectory">
+  <Target Name="BuildHelixCommands">
+    <!-- We need to do this in a target in order to be able to transform package restore sources item into a string -->
+    <PropertyGroup Condition="'$(HelixCommand)' == '' AND '$(IsPackageTesting)' == 'true'">
+      <HelixPreCommands>set DOTNET_CLI_TELEMETRY_OPTOUT=1;set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1;set DOTNET_MULTILEVEL_LOOKUP=0</HelixPreCommands>
```

Comment:
Won't you want this to happen on not-Package-testing as well? or are those set in RunTests.cmd/sh?

Reply:
I think these are not needed for RunTests.cmd/sh because we use our custom built runtime and not the bootstrapped sdk. In this case we're using the full SDK for package testing.